adapters/mobfoxpb/mobfoxpbtest/supplemental/string_imp_ext.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Is this one any different from `imp_ext_string.json`? Perhaps it was duplicated by mistake?

Reply:
Umm.. I see both of them having `ext` set as an empty string.

Ref: https://github.com/prebid/prebid-server/pull/1585/files#diff-068bd5de743e84f0ea42bf24b78266a42b133d1c56746f07f05e29250e53b9c6R20, https://github.com/prebid/prebid-server/pull/1585/files#diff-b3db1ea0f32285de8f539bf8eab4dfce6a8ccbba76a29d79a1fc9756616428caR20

`imp_ext_empty_object.json` and `empty_imp_ext.json` also look similar. They are the ones that have `bid.Ext` set as an empty object rather than an empty string

Ref: https://github.com/prebid/prebid-server/pull/1585/files#diff-fcc47bc4b52242c11553e49496206a1c37641d1c3e62d97bce2c6d130b38499dR20, https://github.com/prebid/prebid-server/pull/1585/files#diff-827dd6469890439efd8f99990038cfd8ca56cb10ac6d886423b5b99d83bb49ceR20

Reply:
Yep, really there was two duplicates